benchmarks/paper/python/warm_trajectory.py

```
@@ -0,0 +1,151 @@
+        for a, b in zip(fresh, warm, strict=True)
+    ]
+    force_errors = [
+        max(
+            abs(float(x) - float(y))
+            for x, y in zip(
+                a["forces_hartree_per_bohr"], b["forces_hartree_per_bohr"], strict=True
+            )
+        )
+        for a, b in zip(fresh, warm, strict=True)
+    ]
+    generator = random.Random(hashlib.sha256(args.trajectory.read_bytes()).hexdigest())
```

Ruff (0.16.1)

[error] Standard pseudo-random generators are not suitable for cryptographic purposes

(S311)